src/ezmsg/util/generator.py

```diff
@@ -56,7 +56,7 @@ def compose(*funcs):
 
 # State class to hold the generator instance
 class GenState(ez.State):
-    gen: Generator
```

Comment:
Default Generator typehint is `Generator[Unknown, None, None]` which caused a lot of linting errors for me using pylance